src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -920,6 +920,15 @@ public bool AreAllOutputsConnected
             get { return outPorts.All(p => p.IsConnected); }
         }
 
+        /// <summary>
+        ///     Are all the inputs of this node disconnected?
+        /// </summary>
+        [JsonIgnore]
+        public bool AreAllInputsDisconnected
```

Comment:
Usually when we design properties, we usually design it the positive ways, e.g. `AreAllInputsConnected`